cfgov/v1/models/learn_page.py

```diff
@@ -301,20 +304,46 @@ class EventPage(AbstractFilterPage):
     def page_js(self):
         return super(EventPage, self).page_js + ['video-player.js']
 
+    def get_venue_coords(self):
+        # Default to Washington DC coordinates
+        venue_coords = '-77.039628,38.898238'
+
+        if not self.venue_city or not self.venue_state:
+            return venue_coords
+
+        location = '{} {}'.format(self.venue_city, self.venue_state)
+        geocode_api_url = 'https://api.mapbox.com/geocoding/v5/mapbox.places/'
```

Comment:
Per the [Mapbox API documentation](https://docs.mapbox.com/api/search/#endpoints), we need to use the `mapbox.places-permanent` endpoint because we are going to store these coordinates in the database. We already use that endpoint elsewhere in the housing counselor app [here](https://github.com/cfpb/cfgov-refresh/blob/61ae49a621dd231cc0902613284df21f64cb58d5/cfgov/housing_counselor/geocoder.py#L175).